gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
I see your point. we do test local version, as you would your test code work outside the lib just like how rspec set up test environment. I think the confusion is `rspec` is set up in a way that your lib behaves like a gem. I personally don't like to use `require_relative` as it needs more maintenance and lengthy unfriendly path.


also what if you tests code in a deeper folder, and you need a lot of `../`, and you decide to restructure your tests code?